ci/check-basics.py

```diff
@@ -88,6 +88,20 @@ def complain(message):
         if path.startswith('demos/thirdparty/'):
             continue
 
+        if path.startswith('tools/accuracy_checker/configs') and path.endswith('.yml'):
+            if (OMZ_ROOT / path).is_symlink():
+                try:
+                    real_path = (OMZ_ROOT / path).resolve(strict=True)
+                    model_name = (OMZ_ROOT / path).stem
+                    if not str(real_path).endswith(model_name + '/accuracy-check.yml'):
+                        complain(f"{path}: should be a symbolic link on accuracy-check.yml from {model_name} model "
+                                 "directory")
+                except:
```

Comment:
What exception are you expecting?

Reply:
I'm trying to check that file to which link is set exists. If the path doesn’t exist and strict is True in path.resolve(), FileNotFoundError is raised.

Reply:
Then you should catch `FileNotFoundError`, not everything.

Reply:
Done